[Synthetics] add CDN mention + complete Resources section (#12477)
DataDog/documentation: content/en/synthetics/browser_tests/test_results.md

A resource corresponds to the combination of requests and assets. 

{{< img src="synthetics/browser_tests/resources_panel.png" alt="Resources Panel"  >}}

> Above the resources tab you can find:

Reviewer: ```suggestion
Above the resources tab you can find the following:
```

- The total step duration time
- The CDN providers serving the resources with a summary of the cache status for each of them

The **Resources** tab shows:

Resource
: The URL of the resource.

CDN

> : The CDN provider that served the resource. Hovering it will showcase the raw cache status.  

Reviewer: ```suggestion
: The CDN provider that served the resource. Hovering it showcases the raw cache status.  
```

> We detect Akamai, Cloudflare, Fastly, Amazon Cloudfront, Netlify, Google Cloud CDN, Imperva, and Sucuri.

Reviewer: ```suggestion
Datadog detects Akamai, Cloudflare, Fastly, Amazon Cloudfront, Netlify, Google Cloud CDN, Imperva, and Sucuri.
```
